<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QDialog" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>583</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>583</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="min_Y">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-6.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="max_Y">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>6.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="max_X">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>6.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="range_Y">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Диапазон Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="range_X">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Диапазон X:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="min_X">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-6.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="replot">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Replot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>dx = </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dx">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>491</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="value">
    <double>0.050000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
